web/ui/react-app/src/ExpressionInput.test.tsx

```diff
@@ -178,7 +178,7 @@ describe('ExpressionInput', () => {
       const downshift = expressionInput.find(Downshift);
       const instance: any = expressionInput.instance();
       const spyBlur = jest.fn();
-      instance.exprInputRef.current = { blur: spyBlur };
```

Comment:
This change is part of https://github.com/prometheus/prometheus/pull/6316

Reply:
Or its here as well because of failing build

Reply:
Whoops shit, added it locally to make tests pass, will remove it from this PR.